stack.yaml

```diff
@@ -1,12 +1,12 @@
 # Match resolver with nixos-unstable haskellPackages
-resolver: lts-19.25
+resolver: lts-20.4
```

Comment:
Should we stick with 19.33? GHC 9.0.2 is still the default on nixpkgs.

Reply:
It was just merged today to be updated to lts-20

Reply:
Awesome!